### PR TITLE
feat: better support for Uint8Array in console

### PR DESCRIPTION
**Pull request by bartlomieju**

### CODE REVIEW

File: core/01_core.js [hunk 1]
```diff
@@ -804,6 +804,8 @@
         typeof arg === "number" || arg === null || arg === undefined
       ) {
         return arg;
+      } else if (arg instanceof Uint8Array) {
```
Comment (petamoriken) [inline core/01_core.js:807]: 🙇 `TypedArrayPrototypeGetSymbolToStringTag(arg) === "Uint8Array"`

File: core/01_core.js [hunk 2]
```diff
@@ -804,6 +804,8 @@
...
+        `Uint8Array(${arg.length}) [${arg.slice(0, 10).join(", ")}]`;
```
Comment (petamoriken) [inline core/01_core.js:808]:
```suggestion
      } else if (TypedArrayPrototypeSymbolToStringTag(arg) === "Uint8Array") {
        `Uint8Array(${TypedArrayPrototypeGetLength(arg)}) [${
          TypedArrayPrototypeJoin(TypedArrayPrototypeSlice(arg, 0, 10), ", ")
        }]`;
```

Comment (bartlomieju) [reply core/01_core.js:808]: Thanks, I remember to use primordials - I just opened a WIP PR so I don't forget I was to update the buffer display.

**Review by mmastrac — APPROVED** [review]
LGTM

**Pull request merged**